src/server/server.js

```diff
@@ -56,12 +57,12 @@ app.use(metricsBundle);
 
 // routes
 app.use('/metrics', trustedNetworks, metricsBundle.metricsMiddleware);
+app.use(generateToken);
 app.use('/', routes.login);
 app.use('/api', routes.github);
 app.use('/api', routes.launchpad);
 app.use('/api', routes.store);
 app.use('/', routes.webhook);
```

Comment:
Maybe move this above `generateToken`?  We don't need or want to generate a new CSRF token for webhook notifications.